scripts/zones/AlTaieu/mobs/Ulxzomit.lua

```diff
@@ -0,0 +1,71 @@
+-----------------------------------
+-- Area: Al'Taieu
+--  Mob: Ul'xzomit
+-----------------------------------
+require("scripts/globals/follow")
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    local mobId = mob:getID()
+    local x = mob:getXPos()
+    local y = mob:getYPos()
+    local z = mob:getZPos()
+    local r = mob:getRotPos()
+
+    if mob:getMobMod(xi.mobMod.LEADER) > 0 then
+        for i = 1, mob:getMobMod(xi.mobMod.LEADER) do
+            local followerId = mobId + i
+            local follower = GetMobByID(followerId)
+
+            if not follower:isSpawned() then
+                local newX = x + math.random(-2, 2)
+                local newY = y
+                local newZ = z + math.random(-2, 2)
+
+                follower:setSpawn(newX, newY, newZ, r)
+                follower:spawn()
+                follower:setMobFlags(1153)
+            end
+
+            xi.follow.follow(follower, mob)
+        end
+    end
+end
+
+entity.onMobEngaged = function(mob, target)
+    local followers = xi.follow.getFollowers(mob)
+
+    if not followers then return end
+
+    for _, follower in ipairs(followers) do
+        if follower:isSpawned() and not follower:isEngaged() then
+            follower:updateEnmity(target)
+        end
+    end
+end
+
+entity.onMobRoam = function(mob, player)
+    -- TODO: Respawn followers?
+    -- They might all stay dead until the entire group is killed, then respawn as a group.
+end
+
+-- Do the following and linking for the babies.
+entity.onMobRoamAction = function(mob)
```

Comment:
see #2629 and comment there, because any change done for that issue will affect the future usage of roaming scripts and I don't want to break your work.